tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -73,7 +74,18 @@ class Meta:
         """Metadata."""
 
         model = User
-        fields = ['id', 'first_name', 'last_name', 'email', 'stt', 'region', 'roles', 'access_request']
+        fields = [
+            'id',
+            'first_name',
+            'last_name',
+            'email',
+            'stt',
+            'region',
+            'roles',
+            'access_request',
```

Comment:
i do believe this could probably be removed from the serializer/model and the frontend at this point, along with `deactivated` but i have not yet